ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host3/router-multicast.yml

```
@@ -0,0 +1,4 @@
+---
+router_multicast:
+  ipv4:
+    multipath: "deterministic"
```

```suggestion
    multipath: deterministic
```
Please update for all places.

---

Done.